KUBEFLOW-OUTREACH-COMMITTEE.md

```diff
@@ -0,0 +1,60 @@
+# Kubeflow Outreach Committee (KOC)
+
+The Kubeflow Outreach Committee (KOC) is a committee dedicated to fostering growth, engagement, and community outreach for the Kubeflow project. It will focus on activities that promote Kubeflow to new and existing users, contributors, and stakeholders while building an inclusive, vibrant, and diverse community within the broader AI/ML ecosystem.
+
+The governance of the Kubeflow Outreach Committee will evolve as the community grows, ensuring that outreach activities remain relevant and impactful.
+
+### Charter
+
+1. Promote the vision, values, and mission of the Kubeflow project.
+2. Lead efforts to grow and engage the AI/ML community, including organizing events, producing educational materials, and advocating for Kubeflow across various tech communities.
+3. Help grow the community by bringing new contributors and users to Kubeflow, particularly through accessible resources and community-building initiatives.
+4. Collaborate with other committees and working groups to align outreach efforts with the broader goals of the Kubeflow project.
+5. Maintain the Kubeflow Website, documentation, demonstration, and educational material for the Kubeflow community.
```

Comment:
What about documentation for Kubeflow Projects which require approval from the WG leads ?

Reply:
I think we should start small in scope.

Reply:
In that case, we might want to be explicit here and say: Maintain the community section in Kubeflow website ?

Reply:
Why not whole website? I think they can be maintainers of the website without having to chase down WG leads. They should work with them of course, but that responsibility is on the WG Lead.

Reply:
As we don't have any "outreach focused" sections of the website right now, I'm not sure there's anything that the KOC could reasonably own.

It's important that the KSC is across any community wide changes to the website, as those pages need to represent the interests of the whole community.

For example, changes to the homepage, about section, and getting started section, should be carefully considered by the KSC, as they impact everyone.

Reply:
Disagree. 

We have real bottlenecks with getting things through on the website already where the difference is arguably marginal to our end users but we end up deliberating over months, which comes at a cost to our users.

Reply:
We are specifically talking about community pages (homepage, about section, getting started section, etc.).

Changes to the community pages __need__ to reflect the interests of the whole community, so it's expected to have significant community discussion around any changes to them. How the project presents itself on community pages impact external investments in the project.

The goal is to get these pages into a state where they don't need to be changed regularly. 

_IDEA: a middle point to these perspectives is to task the KOC with ensuring the pages remain neutral and up-to-date by fostering discussion around proposed changes. However, the final call for any changes would still be up to a vote by the KSC (our only representative elected body)._

Reply:
I like that.

Reply:
I think "maintain the website" it's a lot of work for a few people. My proposal is to contribute to the Website in terms of outreach(events, community information), education activities. 
However, if you want to give the KOC the responsibility to make decisions on usability, accessibility and unified view/user experience about the website, it could be a good thing. and this committee could act when community do not reach a common agreement.

Reply:
>However, if you want to give the KOC the responsibility to make decisions on usability, accessibility and unified view/user experience about the website, it could be a good thing. and this committee could act when community do not reach a common agreement.

This was largely what I invisioned.